### PR TITLE
add package iperf@1.0.13

### PR DESCRIPTION
**Pull request by darmac**
This PR is to add a iperf package in spack.

### CODE REVIEW

File: var/spack/repos/builtin/packages/iperf/package.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# them, you can save this file and test your package like this:
+#
+#     spack install iperf
+#
+# You can edit this file again by typing:
+#
+#     spack edit iperf
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/iperf/package.py]: Delete these comments below the license

File: var/spack/repos/builtin/packages/iperf/package.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    """
+
+    homepage = "https://sourceforge.net/"
+    url      = "https://sourceforge.net/projects/iperf2/files/iperf-2.0.13.tar.gz/download"
+
+    version('2.0.13', sha256='c88adec966096a81136dda91b4bd19c27aae06df4d45a7f547a8e50d723778ad')
+
+    def configure_args(self):
+        args = []
+        return args
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/iperf/package.py]: Delete this function if you aren't going to use it.

Comment (darmac) [reply var/spack/repos/builtin/packages/iperf/package.py]: OK, got it, Thanks.

**Comment by adamjstewart** [comment]
You don't have to close the PR, you can just push another commit to this same branch.

**Comment by darmac** [comment]
A new PR is requested in https://github.com/spack/spack/pull/13816 .

**Pull request closed without merging**